conda/meta.yaml

```diff
@@ -11,34 +11,37 @@ build:
   noarch: python
   preserve_egg_dir: True
```

Comment:
Not sure this is yet mandatory. I will remove it if we do not care to have eggs...